Cargo.toml

```diff
@@ -12,7 +12,7 @@ homepage = "https://github.com/cloud-hypervisor/cloud-hypervisor"
 # Keep in sync with version in .github/workflows/build.yaml
 # Policy on MSRV (see #4318):
 # Can only be bumped by:
-# a.) A dependency requires it, 
+# a.) A dependency requires it,
```

Comment:
nit: unrelated change?

Reply:
I separate the changes of removing whitespaces into its own commit. PTAL.